packages/seller-client/src/components/CopyableTextField/CopyableTextField.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useRef } from 'react';
+import { TextField, IconButton } from '@material-ui/core';
+import { Assignment, Check } from '@material-ui/icons';
+
+import type { TextFieldProps } from '@material-ui/core';
+import useCopyToClipboard from '../../utils/hooks/CopyToClipboard';
```

Comment:
We can use absolute imports in the `seller-client` app like so
```suggestion
import useCopyToClipboard from 'utils/hooks/CopyToClipboard';
```